ExploreASL_Initialize.m

```diff
@@ -1,16 +1,10 @@
-function [x] = ExploreASL_Initialize(DataParPath, ProcessData, iWorker, nWorkers)
+function [x] = ExploreASL_Initialize(varargin)
```

Comment:
Same here - have you changed the calls of ExploreASL_Initialize everywhere if they match to this input parameter change?

Reply:
Same answer as above :)

Reply:
Hopefully fixed all in c70a671.